applications/DeepSpeed-Chat/setup.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+# DeepSpeed Team
+
+# setup.py: install script for deepspeed_chat
+"""
+to install deepspeed_chat and its dependencies for development work,
+run this cmd from the root directory:
+    pip install -e .
+"""
+import setuptools
+
+setuptools.setup(
+    name="deepspeed-chat",
```

Comment:
@mrwyattii - looks like we are thinking of making this into a python package, should we add scripts to publish this then?